owlbot.py

```diff
@@ -30,16 +30,26 @@
       "multiclusteringress",
     ]
 
-    # rename dependencies google.cloud.gkehub.dep.vX to google.cloud.gkehub.dep_vX
     for dep in dependencies:
+        # Copy v1 dependencies from google/cloud/gkehub to google/cloud/gkehub_vX
+        s.copy(library / f"google/cloud/gkehub/{dep}_v1", library / f"google/cloud/gkehub_{library.name}/{dep}_v1")
+        s.copy(library / f"docs/{dep}_v1", library / f"docs/gkehub_{library.name}/{dep}_v1")
```

Comment:
Ah, here is where you made that choice.  I could see leaving the docs files in their original locations (to avoid breaking URLs to them), but am +0 with this change.